lte/gateway/python/magma/mobilityd/main.py

```diff
@@ -9,16 +9,26 @@
 
 from magma.common.service import MagmaService
 from magma.mobilityd.rpc_servicer import MobilityServiceRpcServicer
+from magma.configuration.exceptions import LoadConfigError
+from magma.configuration.service_configs import load_service_config
 from lte.protos.mconfig import mconfigs_pb2
 
 
 def main():
     """ main() for MobilityD """
     service = MagmaService('mobilityd', mconfigs_pb2.MobilityD())
 
+    # Load redis_config
+    try:
+        redis_config = load_service_config('redis')
+    except LoadConfigError as e:
+        logging.warning('Warning: failed to retrie redis config ',
+            'service will fallback to default configuration')
+        redis_config = {}
```

Comment:
What should be the expected behaviour if this redis.yml fails to load?

Right know, if `persist_to_redis` is set it will use the  `localhost:6380` defined in default parameters args.

Reply:
Yeah this should be the expected behavior (the same as creating the default client)